<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimeRFEUSBDialog</class>
 <widget class="QDialog" name="LimeRFEUSBDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LimeRFE USB control</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Dismiss dialog</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="deviceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dev</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openDevice">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Open device</string>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeDevice">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Close Device</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QComboBox" name="device">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Device</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deviceToGUI">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Transfer data to GUI</string>
   </property>
   <property name="text">
    <string>to GUI</string>
   </property>
  </widget>
  <widget class="Line" name="rxSeparator">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="rxChannelLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Rx channel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rxChannelGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Rx channel group</string>
   </property>
   <item>
    <property name="text">
     <string>Wideband</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HAM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cellular</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="rxChannel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Rx channel range</string>
   </property>
  </widget>
  <widget class="QLabel" name="rxPortLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Rx port</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rxPort">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Rx port</string>
   </property>
   <item>
    <property name="text">
     <string>Tx/Rx (J3)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tx/Rx HF (J5)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="attenuationLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Att</string>
   </property>
  </widget>
  <widget class="QComboBox" name="attenuation">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Rx attenuation</string>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="attenuationUnits">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>21</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="amFmNotchFilter">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>AM/FM notch filter</string>
   </property>
   <property name="text">
    <string>Notch</string>
   </property>
  </widget>
  <widget class="Line" name="txSeparator">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="txChannelLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Tx channel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="txFollowsRx">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Channel same as Rx</string>
   </property>
   <property name="text">
    <string>Same as Rx</string>
   </property>
  </widget>
  <widget class="QComboBox" name="txChannelGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Tx channel group</string>
   </property>
   <item>
    <property name="text">
     <string>Wideband</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HAM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cellular</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="txChannel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Tx channel range</string>
   </property>
  </widget>
  <widget class="QLabel" name="txPortLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Tx port</string>
   </property>
  </widget>
  <widget class="QComboBox" name="txPort">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Tx port</string>
   </property>
   <item>
    <property name="text">
     <string>Tx/Rx (J3)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tx (J4)</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="powerSeparator">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="modeRx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Switch Rx</string>
   </property>
   <property name="text">
    <string>RX</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="modeTx">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Switch Tx</string>
   </property>
   <property name="text">
    <string>TX</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="modeText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Rx/Tx state</string>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="modeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Apply changes</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="statusText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Messages</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rxTxToggle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Rx/Tx toggle</string>
   </property>
   <property name="text">
    <string>Toggle</string>
   </property>
  </widget>
  <widget class="QLabel" name="powerFwdLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fwd</string>
   </property>
  </widget>
  <widget class="QLabel" name="powerRefLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref</string>
   </property>
  </widget>
  <widget class="QLabel" name="powerRefText">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>35</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Relative reflected power in dB</string>
   </property>
   <property name="text">
    <string>00.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="powerFwdText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>35</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Relative forward power in dB</string>
   </property>
   <property name="text">
    <string>00.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="powerUnits">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>25</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>dB</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="returnLossLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RL</string>
   </property>
  </widget>
  <widget class="QLabel" name="returnLossText">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>35</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Return loss in dB</string>
   </property>
   <property name="text">
    <string>00.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="powerEnable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enable power measurements</string>
   </property>
   <property name="text">
    <string>Pwr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="powerRefresh">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Refresh power</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/recycle.png</normaloff>:/recycle.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="swrLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>45</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>VSWR</string>
   </property>
  </widget>
  <widget class="Line" name="modeSeparator">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="ButtonSwitch" name="powerAutoRefresh">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Auto refresh power</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/play.png</normaloff>:/play.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="swrText">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>50</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Voltage Standing Wave Ratio</string>
   </property>
   <property name="text">
    <string>1.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="powerSource">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Power measurement source (EXTernal, CELlular)</string>
   </property>
   <item>
    <property name="text">
     <string>EXT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CEL</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="powerCorrLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Corr</string>
   </property>
  </widget>
  <widget class="QLabel" name="powerAbsDbText">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>75</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Corrected forward power in dBm</string>
   </property>
   <property name="text">
    <string>-00.0 dBm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="powerAbsWText">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Corrected forward power in Watts</string>
   </property>
   <property name="text">
    <string>0.000 W</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="powerCorrValue">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="toolTip">
    <string>Power correction in dBm</string>
   </property>
   <property name="text">
    <string>-00.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="powerAbsAvg">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>50</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Corrected power averaging</string>
   </property>
   <property name="text">
    <string>Avg</string>
   </property>
  </widget>
  <widget class="QComboBox" name="deviceSetRx">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Index of Rx DeviceSet </string>
   </property>
  </widget>
  <widget class="QComboBox" name="deviceSetTx">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Index of Tx DeviceSet </string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceSetRxLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rx</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceSetTxLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tx</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="deviceSetSync">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>DeviceSet synchronization</string>
   </property>
   <property name="text">
    <string>Rx/Tx Sync</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deviceSetRefresh">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Refresh DeviceSet indexes</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/recycle.png</normaloff>:/recycle.png</iconset>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LimeRFEUSBDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LimeRFEUSBDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
